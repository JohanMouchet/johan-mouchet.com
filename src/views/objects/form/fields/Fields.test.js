import React from "react";
import { shallow } from "enzyme";
import { Field, Select, Textarea } from ".";

it("Renders <Field /> without crashing", () => {
  shallow(<Field />);
});

it("Renders <Select /> without crashing", () => {
  shallow(<Select />);
});

it("Renders <Textarea /> without crashing", () => {
  shallow(<Textarea />);
});
