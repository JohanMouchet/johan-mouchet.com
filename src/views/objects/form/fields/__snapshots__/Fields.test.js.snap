// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Field /> without crashing 1`] = `
<input
  className="o-form__field o-form__field--undefined"
/>
`;

exports[`Renders <Select /> without crashing 1`] = `""`;

exports[`Renders <Textarea /> without crashing 1`] = `
<textarea
  className="o-form__field o-form__field--textarea"
/>
`;
